Author: Aurelien Jarno <aurel32@debian.org>

Since bochsbios is built with gcc-4.2, the delay loop that is used to
wait for processors to be brought up is being optimized out. This result
in -smp not working in QEMU.

---
 bios/rombios32.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/bios/rombios32.c
+++ b/bios/rombios32.c
@@ -369,7 +369,7 @@ void bios_printf(int flags, const char *
 
 void delay_ms(int n)
 {
-    int i, j;
+    volatile int i, j;
     for(i = 0; i < n; i++) {
 #ifdef BX_QEMU
         volatile int k;
